lib/src/widgets/custom_image_crop_widget.dart

```diff
@@ -67,6 +73,8 @@ class CustomImageCrop extends StatefulWidget {
     this.shape = CustomCropShape.Circle,
     this.cropPercentage = 0.8,
     this.drawPath = DottedCropPathPainter.drawPath,
+    this.canRotate = false,
```

Comment:
For consistency I would default this to true, so that updating the package doesn't change the functionality.

Reply:
sure